design.csv

```diff
@@ -0,0 +1,3 @@
+sample,replicate,fastq_1,fastq_2

```

Comment:
Yup, thats fine - I suppose you're loading that `design.csv` in your ATACseq pipeline then?